scripts/zephyr_module.py

```diff
@@ -758,6 +758,9 @@ def parse_modules(zephyr_base, manifest=None, west_projs=None, modules=None,
 
     if extra_modules is None:
         extra_modules = []
+        for var in ['EXTRA_ZEPHYR_MODULES', 'ZEPHYR_EXTRA_MODULES']:
```

Comment:
both are allowed?

Reply:
`ZEPHYR_EXTRA_MODULES` is the old way of doing it, but it's still being used?